KBase migration - Grafana tutorial (#186)
timescale/docs: timescaledb/tutorials/grafana/installation.md

# Set up TimescaleDB and Grafana

> This tutorial uses Managed Service for TimescaleDB to set up your database, and
> to set up Grafana. You can [create a free account][mst-login] to try it out.

Reviewer: ```suggestion
This tutorial uses Timescale Cloud to set up your database, and
to set up Grafana. You can [create a free account][mst-login] to try it out.
```

## Create a new service for Grafana

> You need to sign in to your Managed Service for TimescaleDB account to create a

Reviewer: ```suggestion
You need to sign in to your Timescale Cloud account to create a
```

new service to run Grafana.

### Procedure: Creating a new service for Grafana

> 1.  [Log in to your MST account][mst-login]. By default, you start in the

Reviewer: ```suggestion
1.  [Log in to your Timescale Cloud account][mst-login]. By default, you start in the
```

1.  In the `Select your cloud service provider` and `Select your cloud service
    region` sections, choose the provider and region that you prefer, or accept
    the default values.
1.  In the `Select your service plan` section, click `Dashboard-1`.
1.  In the `Provide your service name` section, type a name for your new
    service. In this example, we've used `grafana-tutorial`.
    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/mst-nameservice-grafana.png" alt="Name the Grafana service"/>
1.  When you are happy with your selections, click `Create Service` to go back  
    to the `Services` view while your service is created. The status indicator
    says `Rebuilding` while the service is created. It is ready for you to use
    when the indicator is green and says `Running`. This usually takes a couple
    of minutes, but different clouds can vary. You can click the service name in
    the list to see more information and make changes.
    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/mst-buildservice-grafana.png" alt="Building the Grafana service"/>

> ## Log in to your MST Grafana service

Reviewer: ```suggestion
## Log in to your Timescale Cloud Grafana service
```

When your service is built, you can log and set up your data services.

> ### Logging in to your MST Grafana service
> 1.  In the [MST account][mst-login] `Services` view, click the name of your new

Reviewer: ```suggestion
### Logging in to your Timescale Cloud Grafana service
1.  In the [Timescale Cloud account][mst-login] `Services` view, click the name of your new
```
